src/Controls/tests/DeviceTests/Elements/Application/ApplicationTests.Android.cs

```diff
@@ -59,24 +86,7 @@ public async Task SoftInputModeSetOnApplicationPropagatesToWindowHandlers()
 
 						handlers.Insert(0, app.Window.Handler);
 
-						// Validate that the Soft Input initializes to AdjustPan
-						Assert.Equal(ASoftInput.AdjustPan, windowHandler.LastASoftInputSet);
```

Comment:
Did you mean to remove this Assert? (AFAICT everything else except for this line was moved)

Reply:
Yea, this is already being validated by the other test

https://github.com/dotnet/maui/pull/14397/files#diff-2f5fa116e225dc8e883f21962e576c8b13855ed2a873633b51b1d2ebebdfcf9bR24

So, it's not necessary to validate here as well.